Sources/Rendering/Core/ImageCPRMapper/example/controller.html

```diff
@@ -0,0 +1,7 @@
+<table>
+  <tr>
+    <td>
+      <select id='centerline' style="width: 100%"></select>
```

Comment:
you could have added the `angle` input instead of an empty `select`

Reply:
The select is filled with the dirrent centerlines in the JS code
It was not trivial but I added an `angle` input